supportedBackends/tes/src/main/scala/cromwell/backend/impl/tes/TesTask.scala

```diff
@@ -125,117 +131,104 @@ final case class TesTask(jobDescriptor: BackendJobDescriptor,
           TaskParameter(
             Option(globDirName),
             Option(fullyQualifiedTaskName + "." + globDirName),
-            tesPaths.storageOutput(globDirectory),
+            Option(tesPaths.storageOutput(globDirectory)),
             tesPaths.containerOutput(containerWorkDir, globDirectory),
-            "Directory",
-            Option(false)
+            Option("DIRECTORY"),
+            None
           ),
           TaskParameter(
             Option(globListName),
             Option(fullyQualifiedTaskName + "." + globListName),
-            tesPaths.storageOutput(globListFile),
+            Option(tesPaths.storageOutput(globListFile)),
             tesPaths.containerOutput(containerWorkDir, globListFile),
-            "File",
-            Option(false)
+            Option("FILE"),
+            None
           )
         )
     }
 
-  val outputs: Seq[TaskParameter] = wdlOutputs ++ standardOutputs
-
-  // TODO all volumes currently get the same disk requirements
-  private val workingDirVolume = runtimeAttributes
-    .dockerWorkingDir
-    .map(path => Volume(
-      Option(path),
-      runtimeAttributes.disk.to(MemoryUnit.GB).amount.toInt,
-      None,
-      path,
-      Option(false)
-    ))
-
-  val volumes = Seq(
-    Volume(
-      Option(tesPaths.callInputsDockerRoot.toString),
-      runtimeAttributes.disk.to(MemoryUnit.GB).amount.toInt,
-      None,
-      tesPaths.callInputsDockerRoot.toString,
-      // inputs in read-only volume
-      Option(true)
-    ),
-    Volume(
-      Option(tesPaths.callExecutionDockerRoot.toString),
-      runtimeAttributes.disk.to(MemoryUnit.GB).amount.toInt,
-      None,
-      tesPaths.callExecutionDockerRoot.toString,
-      Option(false)
-    )
-  ) ++ workingDirVolume
+  val outputs: Seq[TaskParameter] = wdlOutputs ++ standardOutputs ++ Seq(commandScriptOut)
```

Comment:
The TES backend is now responsible for creating this file so if we want it to end up in the final workflow output directory (or gs bucket) it needs to be an output.